### PR TITLE
[Example] Use L4 as an example

### PR DESCRIPTION
**Pull request by Michaelvll**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: llm/llama-2/README.md [hunk 1]
```diff
@@ -55,9 +55,11 @@ sky launch -c llama-serve -s chatbot-hf.yaml
 
 3. **Optional**: Try other GPUs:
 ```bash
-sky launch -c llama-serve-v100 -s chatbot-hf.yaml --gpus V100:8
+sky launch -c llama-serve-l4 -s chatbot-hf.yaml --gpus L4
 ```
 
+Here L4 is a GPU that is cost-effective for inference. You can find more details about the GPU types [here](https://cloud.google.com/blog/products/compute/introducing-g2-vms-with-nvidia-l4-gpus).
```
Comment (infwinston) [inline llm/llama-2/README.md:61]: is the below description better?

L4 is the latest generation GPU built for large inference AI workloads. Find more details [here](https://cloud.google.com/blog/products/compute/introducing-g2-vms-with-nvidia-l4-gpus).

**Review by infwinston — APPROVED** [review]
LGTM

**Comment by infwinston** [comment]
sorry one thing. let's specify memory requirement in yaml. otherwise the cheapest L4 can't load 7b model

File: llm/llama-2/README.md (hunk 1 above)
Comment (Michaelvll) [reply llm/llama-2/README.md]: Sounds good! Updated.

**Pull request merged**